runtime/lua/vim/lsp/handlers.lua

```diff
@@ -7,6 +7,8 @@ local buf = require 'vim.lsp.buf'
 
 local M = {}
 
+local ns = api.nvim_create_namespace("lsp/popup")
```

Comment:
I'd recommend making `ns` more descriptive, and `lsp/popup` is ambiguous, maybe `lsp/signature_help`

Reply:
I don't really know what that namespace is used for. I'll change names to something more descriptive.

Reply:
lsp_highlight_ns for example

Reply:
fixed